src/execution/__tests__/abstract-test.ts

```diff
@@ -42,6 +42,7 @@ async function executeQuery(args: {
 
 class Dog {
   name: string;
+
```

Comment:
strange, why do you need empty line here?
what `eslint` rule is triggered by it?

Reply:
I might have enabled incorrectly. Now should be updated